packages/gdl-frontend/components/Layout/index.js

```diff
@@ -67,7 +139,81 @@ class Layout extends React.Component<Props, State> {
           onClose={() => this.setState({ drawerIsOpen: false })}
           isOpen={this.state.drawerIsOpen}
         />
-        {wrapWithMain ? <Main component="main">{children}</Main> : children}
+        {wrapWithMain ? (
+          <Main square component="main">
+            {children}
+          </Main>
+        ) : (
+          children
+        )}
+        <Container size="large" stickToEdgeOnLargeScreens width="100%">
+          <Footer>
+            <div
+              css={[
+                {
+                  order: 2,
+                  border: 'solid 1px #e3e3e3',
+                  width: '100%',
+                  margin: '10px'
+                },
+                media.tablet({ display: 'none' })
+              ]}
+            />
+
+            <LinkGrid>
+              <a href="https://home.digitallibrary.io/the-global-digital-library-uses-cookies/">
+                <Trans>Cookie policy</Trans>
+              </a>
+              <a href="https://home.digitallibrary.io/privacy/">
+                <Trans>Privacy policy</Trans>
+              </a>
+              <a href="https://blog.digitallibrary.io/cc/">
+                <Trans>Licensing and reuse</Trans>
+              </a>
+              <a href={config.zendeskUrl}>
+                <Trans>Report issues</Trans>
+              </a>
+
+              <a href="https://home.digitallibrary.io/about/">
+                <Trans>About</Trans>
+              </a>
+              <a href="https://blog.digitallibrary.io/">
+                <Trans>Blog</Trans>
+              </a>
+            </LinkGrid>
+
+            <CreativeCommons>
+              <a
+                href="https://creativecommons.org/"
+                component="a"
+                aria-label="Creative Commons"
+              >
+                <CCLogo css={{ width: '100px' }} />
+              </a>
+            </CreativeCommons>
+
+            <SocialMediaIcons>
+              <a
+                aria-label="Facebook"
+                href="https://nb-no.facebook.com/globaldigitallibrary/"
+              >
+                <FaFacebookSquare />
+              </a>
+              <a
+                aria-label="Twitter"
+                href="https://www.facebook.com/globaldigitallibrary/"
+              >
+                <FaTwitterSquare />
+              </a>
+              <a
```

Comment:
På desktop ser det ut som det er noe margin til høyre for det siste sosiale medier ikonet, mens det ikke er det på venstre side av footeren.